tests/smoke-test-general.bats

```diff
@@ -350,4 +350,35 @@ services:
 	[[ $(echo "$output" | grep -c "feaTures.Alpha-beta_zulu.docksal") -eq 0 ]]
 	[[ "${output}" =~ "The VIRTUAL_HOST has been modified from feaTures.Alpha-beta_zulu.docksal to features.alpha-beta-zulu.docksal to comply with browser standards." ]]
 	unset output
-}
\ No newline at end of file
+}
+
+@test "fin share" {
+	#[[ $SKIP == 1 ]] && skip
+
+        # Send all mail to /bin/true
+        echo "sendmail_path=/bin/true" >> .docksal/etc/php/php.ini
+        # Initialize the Project
+        fin init
+	# Run fin share in a emulated terminal
+	screen -S testNgrok -d -m fin share
+        # sleep so ngrok can load
+        sleep 10
+	# Query API for information
+	API=$(docker exec -it "drupal8_web_1_ngrok" sh -c "wget -qO- http://localhost:4040/api/tunnels")
+	# Return Public URL for site.
+	PUBLIC_HTTP_URL=$(echo "${API}" |  python -c 'import json,sys;obj=json.load(sys.stdin);print obj["tunnels"][0]["public_url"]')
```

Comment:
Is this a Python json parser? If so, I suggest we use (and standardize on) [jq](https://stedolan.github.io/jq/) across the board. There are plans to include both jq and yq in `docksal/cli`

Reply:
@lmakarov until we install jq within the container I'm using the standard Python json parser. I agree that if we can in fact get jq and yq installed within container then we can swap these out and may even want to make note of it for future. The problem is that ngrok previously broke and while 100% test coverage seems far away at the moment. I want to get 1 test in for `fin share` right now. Seeing as there have been individuals who made a note about it and more importantly because of @achekulaev recent blog post about `fin share`

Reply:
`jq` is already there in Travis.

```
echo "${API}" | jq -r '.tunnels[0].public_url'
https://7c1b4c0a.ngrok.io
```

Reply:
completely misunderstood and forgot the jq was being ran in travis and not the container. made the change